multispecies_whale_detection/examplegen.py

```diff
@@ -244,12 +248,16 @@ def make_relative(self,
     Raises:
       ValueError if clip_metadata.start_utc is None.
     """
-    begin, end = _restrict_to_clip(
+    endpoints = _restrict_to_clip(
```

Comment:
(I suppose you could merge the _restrict_to_clip logic into this function, since they're both small functions and _restrict _to_clip doesn't seem to be used elsewhere?)

Reply:
It's used for both UTCAnnotation and FileAnnotation, the only real difference being the type of time (datetime vs. timedelta, which means relative to a file.)

This does make me realize that the thing I've done twice here can move into restrict_to_clip.